templates/repo/pulls/fork.tmpl

```diff
@@ -9,6 +9,13 @@
 				</h3>
 				<div class="ui attached segment">
 					{{template "base/alert" .}}
+
+					{{if not .CanForkRepo}}
+						<div class="ui negative message">
+							<p>{{.locale.TrN .MaxCreationLimit "repo.form.reach_limit_of_creation_1" "repo.form.reach_limit_of_creation_n" .MaxCreationLimit}}</p>
```

Comment:
Surprisingly, your fix wasn't what I meant initially, and rather confused me, even though I see the value in your addition.

What I meant is basically the other direction:
This code here in the template always assumes that an error means that we reached the limit of repos. But I don't think this must necessarily be the case.